### PR TITLE
locale specific variants and amounts for Australia

### PR DESCRIPTION
**Pull request by pvighi**

### CODE REVIEW

File: frontend/app/views/support/ABTest.scala [hunk 1]
```diff
@@ -6,44 +6,77 @@ import java.net.URLEncoder
...
-    Variant("Amount - 40 highlight","40",0,views.html.fragments.giraffe.contributeAmountButtons(List(20,40,75,100),Some(40)))
+    Variant("Amount - 5 highlight","5",0,views.html.fragments.giraffe.contributeAmountButtons(List(5,25,50,100),Some(5)),notAustralia),
+    Variant("Amount - 25 highlight","25",0,views.html.fragments.giraffe.contributeAmountButtons(List(5,25,50,100),Some(25)), notAustralia),
+    Variant("Amount - no highlight","None",0,views.html.fragments.giraffe.contributeAmountButtons(List(5,25,50,100), None), notAustralia),
+    Variant("Amount - 35 highlight","35",0,views.html.fragments.giraffe.contributeAmountButtons(List(10,35,65,100),Some(35)), notAustralia),
+    Variant("Amount - 35 highlight descending","35-descending",0,views.html.fragments.giraffe.contributeAmountButtons(List(100,65,35,10),Some(35)), notAustralia),
+    Variant("Amount - 100 highlight","100",0,views.html.fragments.giraffe.contributeAmountButtons(List(25,50,100,250),Some(100)), notAustralia),
+    Variant("Amount - 50 highlight","50",1,views.html.fragments.giraffe.contributeAmountButtons(List(25,50,100,250),Some(50)), notAustralia),
```
Comment (pvighi) [inline frontend/app/views/support/ABTest.scala]: This is the only pre existing variant with weight > 0 but just for clarity I added the "not australia filter to all of them"

File: frontend/app/views/support/ABTest.scala [hunk 2]
```diff
@@ -6,44 +6,77 @@ import java.net.URLEncoder
...
+    Variant("Amount - 15","15",0,views.html.fragments.giraffe.contributeAmountButtons(List(15,35,65,100),Some(35)), notAustralia),
+    Variant("Amount - 40 highlight","40",0,views.html.fragments.giraffe.contributeAmountButtons(List(20,40,75,100),Some(40)), notAustralia),
+
+    Variant("Amount - 100 highlight Australia", "40-Australia", 1, views.html.fragments.giraffe.contributeAmountButtons(List(50, 100, 250, 500), Some(100)), Set(Australia))
```
Comment (pvighi) [inline frontend/app/views/support/ABTest.scala]: I don't know what the convention is for the variant name and slug I just added Australia to the end of it

Comment (markjamesbutler) [inline frontend/app/views/support/ABTest.scala]: Rename to 100-Australia

File: frontend/app/views/support/ABTest.scala (hunk 1 above)
Comment (markjamesbutler) [inline frontend/app/views/support/ABTest.scala]: Remove all weights at zero.

**Comment by markjamesbutler** [comment]
👍

**Comment by AWare** [comment]
@markjamesbutler: did you want those variants deleting? i'm not sure the zero weighted ones were to be deleted.

File: frontend/app/controllers/Giraffe.scala [hunk 3]
```diff
@@ -28,7 +28,11 @@ object Giraffe extends Controller {
 
 
   val chargeId = "charge_id"
-  val maxAmount: Option[Int] = 2000.some
+
+  def maxAmount(currency: Currency): Option[Int] = currency match {
+    case CountryGroup.Australia.currency => 3500.some
```
Comment (pvighi) [inline frontend/app/controllers/Giraffe.scala]: we still need to confirm what the actual max amount should be in australian dollars

**Comment by markjamesbutler** [comment]
👍  - LGTM

**Comment by pvighi** [comment]
Tested

**Pull request merged**